pkgs/by-name/ir/ironbar/package.nix

```diff
@@ -17,20 +17,26 @@
 , rustPlatform
 , lib
 , fetchFromGitHub
+, luajit
+, luajitPackages
+, libpulseaudio
+, features ? []
```

Comment:
This isn't actually changing the features provided to the rust builder

Reply:
This in itself no, however my understanding is that this would provide an attribute that users can override which is then used by the `hasFeature` function used in `buildInputs` and gappsWrapperArgs

Reply:
Unless you are wanting to be able to use `features` the same way cargo uses them when defining dependencies in Cargo.toml, however I'm not 100% on how to do that with `rustPlatform` although could probably investigate. 

This is based off of what was provided by upstreams `default.nix`

Reply:
You can tell the builder to use a defined set of features like so:
```rs
buildNoDefaultFeatures = true;
buildFeatures = [  ];
```
The way the upstream one does it, is it checks if the provided features list is empty, if it is empty then it will enable default features, otherwise it disables default features and provides the features.

Reply:
IMO it doesn't make sense to provide a features list, if it doesn't actually like do anything

Reply:
I'll add both `buildNoDefaultFeatures = true` and `buildFeatures = features`.